substrate/client/network/src/protocol.rs

```diff
@@ -376,6 +376,27 @@ impl<B: BlockT> NetworkBehaviour for Protocol<B> {
 					)
 				}
 			},
+
+			NotificationsOut::ProtocolMisbehavior { peer_id, set_id } => {
+				let index: usize = set_id.into();
+				let protocol_name = self.notification_protocols.get(index);
+
+				error!(
```

Comment:
```suggestion
				debug!(
```

What should the operator do with this line?